test/fgpu/test_recv.py

```diff
@@ -82,14 +84,17 @@ def data_ringbuffer(layout) -> spead2.recv.asyncio.ChunkRingbuffer:
 
 
 @pytest.fixture
-def free_ringbuffer(layout) -> spead2.recv.asyncio.ChunkRingbuffer:
+def free_ringbuffer(layout) -> spead2.recv.ChunkRingbuffer:
     """Create asynchronous free chunk ringbuffer, to be used by the receive streams."""
-    return spead2.recv.asyncio.ChunkRingbuffer(4)
+    return spead2.recv.ChunkRingbuffer(4)
 
 
 @pytest.fixture
 def stream_group(
-    layout, data_ringbuffer, free_ringbuffer, queue
+    layout: Layout,
+    data_ringbuffer: spead2.recv.asyncio.ChunkRingbuffer,
+    free_ringbuffer: spead2.recv.ChunkRingbuffer,
+    queue: spead2.InprocQueue,
```

Comment:
So the `free_ringbuffer` is blocking/synchronous, but the `data_ringbuffer` is still asynchronous? Maybe I should look at the whole test.